libzeth/test/powersoftau_test.cpp

```diff
@@ -15,10 +16,13 @@ using G1 = libff::G1<ppT>;
 using G2 = libff::G2<ppT>;
 using namespace libsnark;
 using namespace libzeth;
+namespace fs = boost::filesystem;
 
 namespace
 {
 
+fs::path g_testdata_dir = fs::path("..") / "testdata";
```

Comment:
Wouldn't it be possible to use an environment variable here? Otherwise we need to assume that the tests are written from a specific location (else, we end up writing `testdata` in the user's filesystem)

Reply:
Oh, just realized that you overwrote this global variable before running the tests..